src/components/styles/NavigationBar.css

```diff
@@ -1,14 +1,11 @@
-* {
-  font-family: "Time New Roman";
-}
-
 .navigationBar {
   color: var(--global-headline-1-color);
   padding: 10px;
   padding-left: 5%;
   padding-right: 2%;
   position: sticky;
   z-index: 1;
+
   background-color: var(--global-page-background-color);
   border: 1px solid var(--secondary);
```

Comment:
Global variables change needed

Reply:
well I changed all the --secondary, --primary stuff into global variables